packages/odyssey-storybook/src/guidelines/DownstreamDevelopment.stories.mdx

```diff
@@ -0,0 +1,313 @@
+    cd odyssey-demo && \
+    git init && \
+    npm init
+```
+
+With your project initialized, it is time to install Odyssey development dependencies:
+
+```sh
+  $ npm install --save-dev \
+      @okta/odyssey-babel-preset \
+      @okta/odyssey-postcss-preset \
+      @okta/odyssey-babel-loader \
+      @okta/browserslist-config-odyssey
+```
+
+Now install foundational development dependencies:
+
+```sh
+  $ npm install --save-dev \
+      typescript \
+      @types/react \
+      @types/react-dom \
+      webpack \
+      webpack-cli \
+      webpack-dev-server \
+      html-webpack-plugin \
+      postcss
+```
+
+And update the `package.json` file at the root of your project:
+
+```json
+"scripts": {
+  "start": "NODE_ENV=development webpack serve --hot --open"
+  "build": "NODE_ENV=production webpack"
+}
+```
+
+### Webpack
+
+Begin by creating a [`webpack.config.js` config file][2] at the root of your project:
+
+```js
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+
+module.exports = {
+  entry: "./src/index.tsx",
+  output: {
+    filename: "bundle.[hash].js",
+    path: path.resolve(__dirname, "dist"),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: "./public/index.html",
+    }),
+  ],
+  resolve: {
+    modules: [path.resolve(__dirname, "./src"), "node_modules"],
+    extensions: ["*", ".js", ".jsx", ".tsx", ".ts"],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx|ts|tsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "@okta/odyssey-babel-loader",
+        },
+      },
+    ],
+  },
+};
+```
+
+[2]: https://webpack.js.org
+
+### Browserslist
+
+Begin by creating a [`.browserslistrc` config file][3] at the root of your project:
+
+```
+extends @okta/browserslist-config-odyssey
+```
+
+Alternatively, add a `browserslist` entry to your package.json file.
+
+This config will allow postcss, babel and other toolchains to optimize
+your build based on the intended browser targets. The configuration provided
+from the `@okta/browserslist-config-odyssey` package mirrors
+[Okta's own published browser support list][4]. You should feel free to
+manipulate this configuration further to optimize your asset delivery.
+For example, to construct separate modern and IE11 only bundles that
+include appropriate polyfills and transpiled source code.
+
+[3]: https://github.com/browserslist/browserslist
+[4]: https://help.okta.com/en/prod/Content/Topics/Miscellaneous/Platforms_Browser_OS_Support.htm
+
+### Babel
+
+Begin by creating a [`babel.config.js` config file][5] the root of your project:
+
+```js
+module.exports = {
+  presets: ["@okta/odyssey-babel-preset"],
+
+  env: {
+    test: {
+      presets: [
+        [
+          "@okta/odyssey-babel-preset",
+          {
+            odyssey: {
+              identityObjectProxy: true,
+            },
+          },
+        ],
+      ],
+    },
+    production: {
+      comments: false,
+    },
+  },
```

Comment:
I might be missing something somewhere else in the docs, but what is `identityObjectProxy`?

Do we want to add a code comment about it in this example?

Reply:
Looks like it's described below, but I still think it might be better inline with the code. Took me a while to find what it did.

Reply:
updated